src/api/routes/tx.ts

```diff
@@ -152,10 +152,28 @@ export function createTxRouter(db: PgStore): express.Router {
           InvalidRequestErrorType.invalid_param
         );
       }
+
+      const orderBy = req.query.order_by;
+      if (orderBy !== undefined && orderBy != 'fee' && orderBy != 'age' && orderBy != 'size') {
```

Comment:
I think we typically put constants like these into an enum or string union type (either a manually defined type, or one generated in the .d.ts from the openapi spec)

Reply:
Done!